concerned-coyotes/earlyinternet/commands/management/commands/run_tasks.py

```diff
@@ -0,0 +1,24 @@
+import importlib
+import time
+
+from django.core.management.base import BaseCommand
```

Comment:
It's a clever approach to use the builtin management system to start your background tasks.